WordPress/src/main/java/org/wordpress/android/ui/accounts/helpers/FetchBlogListWPCom.java

```diff
@@ -44,6 +44,9 @@ protected void fetchBlogList(final Callback callback) {
                     site.put("isAdmin", jsonSite.get("user_can_manage"));
                     site.put("isVisible", jsonSite.get("visible"));
 
+                    // store capabilities as a text blob
+                    site.put("capabilities", jsonSite.get("capabilities").toString());
```

Comment:
I see what you mean. Notice though that the rest of the `.get` usages don't need a null check as they are only putting the value in the HashMap (which allows nulls), while in this case the code tries to access `.toString()`. Even if you see the server sending the 'capabilities' field, unless it is documented to always include a non-null list we better guard for it.

Alternatively, you can use `jsonSite.getString()` directly to workaround both the null check and the `.toString()` call.